content/ytt/docs/latest/lang-ref-string.md

```diff
@@ -127,3 +127,29 @@ Copied here for convenience from [Starlark specification](https://github.com/goo
 ```python
 "Hello, World!".upper()  # "HELLO, WORLD!"
 ```
+
+* [multiline string literal](https://github.com/google/starlark-go/blob/master/doc/spec.md#string-literals)
+  may spread over multiple source lines. It is denoted using three quotation marks at start and end. Within it, unescaped newlines and quotation marks (or even pairs of quotation marks) have their literal meaning, but three quotation marks end the literal.
+  
+  ytt uses a slightly modified version of Starlark that does not preserve indentation in muli-line strings. There is an [issue](https://github.com/vmware-tanzu/carvel-ytt/issues/327) to capture this, and as a workaround we recommend using a [`yamlfragment`](lang-ref-yaml-fragment.md) described below.
```

Comment:
this is only true when working with .yml files. .star files dont have this problem afaik.

Reply:
Could you give me an example of how you can use a .star file without rendering them using a .yml file template? I'm not able to find a case where this works for .star files.

Reply:
funcs.star in https://carvel.dev/ytt/#example:example-load is in starlark format, so you can use multiline strings there the starlark way.

Reply:
I see the output is still showing the same problem of not respecting indentation even when using a `.star` file. https://carvel.dev/ytt/#gist:https://gist.github.com/cari-lynn/66699b6c7e759947fabb55c261d15fb9